ApprovalTests/namers/ApprovalTestNamer.h

```diff
@@ -109,6 +109,11 @@ R"(* Welcome to Approval Tests.
 *     #define APPROVALS_DOCTEST
 *     #include "ApprovalTests.hpp"
 *
+* To do this in UT, add the following to your main.cpp:
```

Comment:
This is brilliant - thank you for updating this.

Trivia for info: this changing is why the CI tests failed - there is an approval test (in dockets tests, I think) that approves the content of this text - so that we can include it in the documentation. Someone will at some point need to re-run that test and approve the new output, then the tests will pass again.